src/client/cli/formatter/csv_formatter.cpp

```diff
@@ -56,10 +56,11 @@ std::string mp::CSVFormatter::format(const ListReply& reply) const
 
     for (const auto& instance : format::sorted(reply.instances()))
     {
-        fmt::format_to(buf, "{},{},{},{},{},\"{}\"\n", instance.name(),
-                       mp::format::status_string_for(instance.instance_status()),
-                       instance.ipv4_size() ? instance.ipv4(0) : "", instance.ipv6_size() ? instance.ipv6(0) : "",
-                       instance.current_release(), fmt::join(instance.ipv4(), ","));
+        fmt::format_to(
+            buf, "{},{},{},{},{},\"{}\"\n", instance.name(), mp::format::status_string_for(instance.instance_status()),
+            instance.ipv4_size() ? instance.ipv4(0) : "", instance.ipv6_size() ? instance.ipv6(0) : "",
+            instance.current_release().empty() ? "Not Available" : fmt::format("Ubuntu {}", instance.current_release()),
```

Comment:
Would this hardcode "Ubuntu" in all output, even if it's a different distro?

Reply:
This information comes from https://cloud-images.ubuntu.com/releases/streams/v1/com.ubuntu.cloud:released:download.json and *only* Ubuntu images are available there.  If one launches a different distro via `https://` or `file://`, we have no insight as to what the distro is, so we use `Not Available`.

Reply:
Ah, okay, so it exclusively hardcodes the printout "Ubuntu" for releases from the stream at ubuntu.com, and will print "Not Available" for anything else. Got it! Thanks for the explanation.

Just to be sure, this is the formatting for `multipass list`, and not `multipass find`?

Also, the information that this is "ubuntu" is available in that `cloud-images.ubuntu.com` stream under the key `os`.

Though, upon launching an image, the `os` key is not stored in `multipassd-vm-instances.json`:

`multipassd-vm-instances.json`
```json
{
    "example": {
        "image": {
            "aliases": [
            ],
            "current_release": "",
            "id": "e1ce033239f0038dca5ef09e582762ba0d0dfdedc1d329bc51bb0e9f5057af9d",
            "initrd_path": "",
            "kernel_path": "",
            "original_release": "22.04 LTS",
            "path": "/var/root/Library/Application Support/multipassd/qemu/vault/instances/example/ubuntu-22.04-server-cloudimg-arm64.img",
            "release_date": "20220712"
        },
        "last_accessed": 1658837383975983,
        "query": {
            "persistent": false,
            "query_type": 0,
            "release": "jammy",
            "remote_name": ""
        }
    }
}
```

Would it be an option to store the `os` key in `multipassd-vm-instances.json` and use that?

Reply:
Hey @larstobi,

> Just to be sure, this is the formatting for `multipass list`, and not `multipass find`?

Yes, this PR **only** applies to `multipass list`.

> Would it be an option to store the `os` key in `multipassd-vm-instances.json` and use that?

Yes, it's possible, but IMO is outside the scope of this PR.  Two things would need to be considered though.  One is to use this current behavior in already existing instances that would be missing the `os` key and the second is to make sure `ubuntu` is capitalized when resolving the `os` key on output.